src/plugins/intel_cpu/src/compiled_model.cpp

```diff
@@ -73,6 +77,16 @@ CompiledModel::CompiledModel(const std::shared_ptr<ov::Model>& model,
     std::vector<Task> tasks;
     tasks.resize(streams);
     m_graphs.resize(streams);
+    auto initScheduler = [&]() {
+#if defined(OV_CPU_WITH_ACL)
+        static std::once_flag flag_once;
+        std::shared_ptr<arm_compute::IScheduler> acl_scheduler = std::make_shared<ACLScheduler>();
+        std::call_once(flag_once, [&]() {
+            arm_compute::Scheduler::set(std::static_pointer_cast<arm_compute::IScheduler>(acl_scheduler));
+        });
+#endif
+    };
```

Comment:
NIT. As we call `initScheduler` in only one place (L99), isn't it better to just inline the implementation there instead of creating a lambda and capturing it in another lambda?